site/src/docs/components/stepper/code.mdx

```diff
@@ -239,15 +239,15 @@ import { StepState, Stepper, Button, IconArrowLeft, IconArrowRight } from 'hds-r
           variant="secondary"
           onClick={() => dispatch({ type: 'setActive', payload: state.activeStepIndex - 1 })}
           style={{ height: 'fit-content', width: 'fit-content' }}
-          iconLeft={<IconArrowLeft />}
+          IconStart={<IconArrowLeft />}
```

Comment:
Suddenly IconStart with capped first letter?  Same on three occasions later in this file.

Reply:
Good spotting